### PR TITLE
Next.timeline

### PR DESCRIPTION
**Pull request by BertrandPodevin**
Add timeline component for LF Next

image
image
image
image
image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1615

File: packages/scss/src/components/timeline/mods.scss [hunk 1]
```diff
@@ -1,6 +1,142 @@
...
+				left: 0;
+			}
+		}
+	}
+
+	#{$description} {
+		padding-left: var(--spacings-big);
+	}
+
+	&.mod-small {
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/timeline/mods.scss:82]: You should not have a mod class statement here.

Comment (vvalentin-lucca) [reply packages/scss/src/components/timeline/mods.scss:82]: When mods are combined, I use a dedicated mixin which I call from the index with
```
&.mod-vertical {
	&.mod-small {
		@include verticalSmall
	}
}
```

File: packages/scss/src/components/util/index.scss [hunk 2]
```diff
@@ -24,6 +24,10 @@
 @include core.palettes;
 @include core.borderRadius;
 
+.#{config.$prefix}positionStatic {
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/util/index.scss:27]: I'll do this again soon via a mixin that is intended to generate this.

**Review by vvalentin-lucca — APPROVED** [review]

**Pull request merged**